core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -1393,27 +1393,31 @@ fn handle_graphics_triangle_path<'gc>(
         culling_to_triangle_culling(activation, culling)?
     };
 
-    let vertices = obj
-        .get_public_property("vertices", activation)?
-        .coerce_to_object(activation)?;
+    let vertices = obj.get_public_property("vertices", activation)?.as_object();
 
-    let indices = obj
-        .get_public_property("indices", activation)?
-        .coerce_to_object(activation)?;
+    let indices = obj.get_public_property("indices", activation)?.as_object();
 
-    let _uvt_data = obj
-        .get_public_property("uvtData", activation)?
-        .coerce_to_object(activation)?;
+    let _uvt_data = obj.get_public_property("uvtData", activation)?.as_object();
```

Comment:
I suppose the change from `coerce_to_object` to `as_object` is to avoid boxing primitives, which would not be valid here anyway?

Reply:
`coerce_as_object` will produce an error if it's `null`, but we want to handle the `null` case here

Reply:
Gotcha.